### ISSUE TITLE
examples in viewHelp documentation return errors

### ISSUE THREAD

**Issue by aarondall** [Macaulay2/M2#987]
Here is code for the suggested examples on the `viewHelp` documentation page.
```
viewHelp -- top of local copy of the documentation, including installed packages
viewHelp "Macaulay2" -- top of Macaulay2 doc
viewHelp ideal -- online doc for the 'ideal' function
viewHelp "matrices" -- overview of matrices in Macaulay2
viewHelp (ideal,List) -- online doc for ideal(List) method
viewHelp (diff,Matrix,Matrix) -- online doc for the diff function taking two matrices as arguments
viewHelp [gb,DegreeLimit] -- view doc for the optional argument DegreeLimit to gb function
viewHelp (symbol**,Matrix,Matrix) -- view doc for Matrix**Matrix
```

In an M2 terminal session I get the following outputs:

```
i1 : viewHelp "viewHelp"

i2 : viewHelp -- top of local copy of the documentation, including installed packages

i3 : viewHelp "Macaulay2" -- top of Macaulay2 doc
stdio:3:1:(3): error: internal error: html documentation file does not exist: /Applications/Macaulay2-1.12/share/doc/Macaulay2/Macaulay2Doc/html/___Macaulay2.html

i4 : viewHelp ideal -- online doc for the 'ideal' function

i5 : viewHelp "matrices" -- overview of matrices in Macaulay2

i6 : viewHelp (ideal,List) -- online doc for ideal(List) method
stdio:6:1:(3): error: no method found for applying -*Function*- to:
     argument 1 :  ideal (of class MethodFunctionSingle)
     argument 2 :  List (of class Type)

i7 : viewHelp (diff,Matrix,Matrix) -- online doc for the diff function taking two matrices as arguments
stdio:7:1:(3): error: no method found for applying -*Function*- to:
     argument 1 :  diff (of class MethodFunction)
     argument 2 :  Matrix (of class Type)
     argument 3 :  Matrix (of class Type)

i8 : viewHelp [gb,DegreeLimit] -- view doc for the optional argument DegreeLimit to gb function

i9 : viewHelp (symbol**,Matrix,Matrix) -- view doc for Matrix**Matrix
stdio:9:1:(3): error: no method found for applying -*Function*- to:
     argument 1 :  ** (of class Keyword)
     argument 2 :  Matrix (of class Type)
     argument 3 :  Matrix (of class Type)
```

All of the corresponding `help` commands work as expected.

**Comment by DanGrayson** [comment]
I've fixed all but one of these in commit d94a3684e274830c7cc3ad788d1d989c868e64b3.  The other one (viewHelp "Macaulay2"), depends on redirecting to the main documentation node, which in this case is "Macaulay2Doc".

**Comment by DanGrayson** [comment]
Something like this would be a good example for the documentation:
```

i10 : elapsedTime nanosleep 500000000
     -- 0.503543 seconds elapsed

o10 = 0
```
